added RF.Guru µPico Board

ure opened the pull request:
request for a pid for our soon to be release open hardware/open source development board

tannewt reviewed (changes requested):
Please include the original ecad files so that they can be modified. Please also link to your CircuitPython branch with your board definition.

ure commented:
> Please include the original ecad files so that they can be modified. Please also link to your CircuitPython branch with your board definition.

Hi @tannewt,

I added a link to the OSHWlab in our README.md (https://github.com/Guru-RF/MicroPico/blob/main/README.md)
I added a temporary link to our fork with the board definition (we will update it later when it is merged into main @ adafruits)

kr,

tannewt reviewed (approved):
Thanks! EasyEDA link with CERN license.
